x-pack/plugins/secops/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -163,7 +164,14 @@ const getHostsColumns = (): Array<Columns<HostsEdges>> => [
     name: i18n.FIRST_SEEN,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => defaultToEmptyTag(node.firstSeen),
+    render: ({ node }) =>
+      node.firstSeen && node.firstSeen !== '' ? (
+        <EuiToolTip position="bottom" content={node.firstSeen}>
+          <FormattedRelative value={node.firstSeen} />
+        </EuiToolTip>
+      ) : (
+        defaultToEmptyTag(node.firstSeen)
+      ),
```

Comment:
We might want to encapsulate this conditional logic into another routine or push it down into `defaultToEmptyTag` if we begin noticing this pattern being repeated in a lot of areas.

No changes requested, I just enjoy the one liner functions we can test when we can write them.